frontend/tests/utils/wmgUtils.ts

```diff
@@ -1,10 +1,25 @@
 import { ROUTES } from "src/common/constants/routes";
 import { TEST_URL } from "../common/constants";
-import { expect, Page } from "@playwright/test";
+import { expect, Page, test } from "@playwright/test";
 import { getTestID, getText } from "tests/utils/selectors";
 import { selectFirstOption, tryUntil } from "./helpers";
 import { ADD_GENE_BTN, ADD_TISSUE_BTN } from "../common/constants";
 
+const { skip } = test;
+
+const ENVS_TO_RUN_TESTS = [
+  "api.cellxgene.dev.single-cell.czi.technology",
+  "api.cellxgene.staging.single-cell.czi.technology",
+  "api.cellxgene.cziscience.com",
+];
+
+export function conditionallyRunTests() {
```

Comment:
this is the function that will use the env variable and check if the API URL is one of the URLs in `ENVS_TO_RUN_TESTS` to determine whether to run the tests or not